components/concept/sync/src/main/java/mozilla/components/concept/sync/Devices.kt

```diff
@@ -0,0 +1,147 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+@file:SuppressWarnings("TooManyFunctions")
+package mozilla.components.concept.sync
+
+import androidx.lifecycle.LifecycleOwner
+import kotlinx.coroutines.Deferred
+import mozilla.components.support.base.observer.Observable
+
+/**
+ * Describes available interactions with the current device and other devices associated with an [OAuthAccount].
+ */
+interface DeviceConstellation : OAuthAccount, Observable<DeviceEventsObserver> {
```

Comment:
This relationship feels wrong to me. Why is a device constellation a `OAuthAccount`? I think you mentioned that in your comment also.